app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -260,11 +260,11 @@ def display_phone_to_deliver_to
 
   def reenter_phone_number_path
```

Comment:
What if we called this method `reenter_phone_number_path_name` since it's no longer the actual path?